### PR TITLE
Refs #29567 - put nulldb requirement into an own group

### PR DESCRIPTION
**Pull request by evgeni**
otherwise our packaging helpers won't pick it up properly

### CODE REVIEW

**Comment by ekohl** [comment]
Relevant discussion: https://github.com/theforeman/foreman/pull/7594/files#r410939636

**Comment by evgeni** [comment]
mhh, yeah, this fails, even tho we don't test that on Jenkins (yet)

if I follow the suggestion and add 'nulldb/rails' to the apipie initializer, it works, but that's surely not what we want.

File: bundler.d/nulldb.rb [hunk 1]
```diff
@@ -1 +1,3 @@
-gem 'activerecord-nulldb-adapter'
+group :nulldb do
+  gem 'activerecord-nulldb-adapter'
```
Comment (ekohl) [inline bundler.d/nulldb.rb:2]: Untested, but does this work?
```suggestion
  gem 'activerecord-nulldb-adapter', require: 'nulldb/rails'
```

Comment (evgeni) [reply bundler.d/nulldb.rb:2]: no

**Comment by evgeni** [comment]
well, THAT works :see_no_evil:

File: bundler.d/nulldb.rb [hunk 2]
```diff
@@ -1 +1,4 @@
...
+end
 gem 'activerecord-nulldb-adapter'
```
Comment (timogoebel) [inline bundler.d/nulldb.rb:4]: Don't we have this twice now?

Comment (evgeni) [reply bundler.d/nulldb.rb:4]: We do, but: the group one is needed for our packaging tooling, and the non-group one needed for nulldb to properly initialize, see the discussion ewoud linked above

Comment (timogoebel) [reply bundler.d/nulldb.rb:4]: Ok, I believe this is one of the rare cases when a comment in the code is in order.

**Pull request closed without merging**